### ISSUE TITLE
Account identification without email / multi identifier

### ISSUE THREAD

**Issue by rolfschmidt** [zammad/zammad#613]
User recommendation:
> It should be possible to create customers without email as mandatory field. Some of our customers dont have a email address ready so we would prefer the phone number as identification for the customer.

Also requested in T#1075669, 1080460, 1086704

**Comment by dennybresch** [comment]
👍

**Comment by zabarabab** [comment]
The incoming call issue is still not resolved: a caller does not necessarily have an email-Address. 

This issue has been raised several times (e.g. #181).

If zammad really needs a contact information, then why not turn it into a url? 

For email, this could be: `mailto:max@provider.com`
For a phone number: `tel:+1-201-555-0123`

**Comment by enskymedia** [comment]
Is there any solution for problems like this yet?

**Comment by Megachip** [comment]
Dear dev team... any ETA or progress? I have the same user 5 times in the DB cause he uses different email addresses to contact us.
That makes everything a bit complicated: stats, find other tickets of that user and so on...

**Comment by MrGeneration** [comment]
This lives in our feature backlog and has no ETA yet.
We will update this issue accordingly if anything moves on this topic.

**Comment by MrGeneration** [comment]
Sorry for the hassle.
After we stumbled upon this issue today we noticed that this functionality already is supported by Zammad.

Actually it's an duplicate of https://github.com/zammad/zammad/issues/181

I'm therefor closing this issue. 🙌

**Comment by thorsteneckel** [comment]
There is more to it. There is currently no way to have a second attribute of e.g. type email (or other "identifier" attribute) that should be used to identify users. Therefore I reopen this issue.

**Comment by MrGeneration** [comment]
You've mentioned this issue in https://github.com/zammad/zammad/issues/272 and closed it as duplicate.
I think this is actually wrong, because the only part this issue mentions "multi identifiers" is its title.

However, issue #272 does mention that alot better in more detail, which is why I think this is a duplicate of above mentioned #181 and thus should be closed.

I'll gladly adjust the title to :x

**Comment by thorsteneckel** [comment]
I agree that we should improve this issue here to meet our latest standards (template, expressive description etc.). It should describe the requirement of having multiple identifiers per user.

**Comment by Megachip** [comment]
<img width="703" alt="image" src="https://user-images.githubusercontent.com/487352/115535921-80d3eb00-a299-11eb-9582-92a88af3edb3.png">
Found this within Zammad, so is it possible for a user to have multiple emails?

Also it is possible to create additional attributes of type email...
<img width="631" alt="image" src="https://user-images.githubusercontent.com/487352/115536152-c2649600-a299-11eb-8aaf-e3458af85bce.png">

**Comment by MrGeneration** [comment]
@Megachip can you *please* stop hijacking unrelated issues and post technical questions to them?

**Comment by YetAnotherGerrit** [comment]
E-Mail is not a mandatory field anymore (don't know since when), so I consider this solved.
